erpnext/selling/utils/order_for.py

```diff
@@ -0,0 +1,36 @@
+import frappe
+from frappe import _dict
+from erpnext.shopping_cart.cart import get_party
+
+def on_session_start():
```

Comment:
- Can we rename `on_session_start` to `initialise_order_for` ??
- It'll give more context and idea what the function does since `on_session_start` is too generic

Reply:
Yeah, sounds good. Will update the PR